src/lib.rs

```diff
@@ -392,6 +402,43 @@ impl Config {
             combined_config.insert(YAML::from(key.clone()), value.clone());
         }
 
+        // Setup large default rate limits.
+        let large_rate_limit: serde_yaml::Value = yaml!({
+            "per_second" => 1_000_000_000,
+            "burst_count" => 1_000_000_000,
```

Comment:
so just as a question, are we sure that we have to provide a ridiculously large number and there's not some other way to turn them off? Just to make sure

Reply:
I just double-checked, there is no way atm.